tests/layer_tests/jax_tests/test_select_n.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import numpy as np
+import pytest
+from jax import numpy as jnp
+
+from jax_layer_test_class import JaxLayerTest
+
+rng = np.random.default_rng(5402)
+
+
+class TestSelectN(JaxLayerTest):
+    def _prepare_input(self):
+        self.case_num = 2 if (self.input_type == np.bool or self.input_type == bool) else self.case_num
+        cases = []
+        which = rng.uniform(0,self.case_num, self.input_shape).astype(self.input_type)
+        which = np.array(which)
+        for i in range(self.case_num):
+            cases.append(jnp.array(rng.uniform(i*10, (i+1)*10, self.input_shape).astype(self.input_type)))
+        cases = np.array(cases)
+        return (which, cases)
+
+    def create_model(self, input_shape, input_type, case_num):
+        self.input_shape = input_shape
+        self.input_type = input_type
+        self.case_num = case_num
+
+        def jax_select_n(which, cases):
+            return jax.lax.select_n(which, *cases)
+        
+        return jax_select_n, None, 'select_n'
+        
+
+    @pytest.mark.parametrize("input_shape", [[],[1],[2,3],[4,5,6],[7,8,9,10]])
+    @pytest.mark.parametrize("input_type", [np.int32, np.int64, bool])
```

Comment:
```suggestion
    @pytest.mark.parametrize("input_type", [np.float32, np.int32, np.int64])
```
let us do not use this type for `which`. We should use this `input_type` only for cases.
Use bool in case_num=2 and np.int32 for which, otherwise.

Reply:
done